<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1642005145501' stop='1642005176465'>
    <name>Verify Login Scenarios on Saxobank React Native Mobile App</name>
    <title>Verify Login Scenarios on Saxobank React Native Mobile App</title>
    <test-cases>
        <test-case start='1642005145502' status='broken' stop='1642005176463'>
            <name>User should be able to login using valid credentials to Saxobank Mobile App</name>
            <title>User should be able to login using valid credentials to Saxobank Mobile App</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='device' value='Pixel 2 API 30 11-11'/>
            </parameters>
            <steps/>
            <attachments/>
            <failure>
                <message>element ("//android.widget.EditText[@resource-id="field_userid"]") still not displayed after 30000ms</message>
                <stack-trace>Error: element ("//android.widget.EditText[@resource-id="field_userid"]") still not displayed after 30000ms
                    at E:\Projects\Test-mobile\MobileAutomationTest\node_modules\webdriverio\build\commands\browser\waitUntil.js:66:23
                    at async Element.wrapCommandFn (E:\Projects\Test-mobile\MobileAutomationTest\node_modules\@wdio\utils\build\shim.js:131:29)
                    at async Element.elementErrorHandlerCallbackFn (E:\Projects\Test-mobile\MobileAutomationTest\node_modules\webdriverio\build\middlewares.js:24:32)
                    at async Element.wrapCommandFn (E:\Projects\Test-mobile\MobileAutomationTest\node_modules\@wdio\utils\build\shim.js:131:29)
                    at async Element.wrapCommandFn (E:\Projects\Test-mobile\MobileAutomationTest\node_modules\@wdio\utils\build\shim.js:131:29)
                    at async Element.elementErrorHandlerCallbackFn (E:\Projects\Test-mobile\MobileAutomationTest\node_modules\webdriverio\build\middlewares.js:24:32)
                    at async Element.wrapCommandFn (E:\Projects\Test-mobile\MobileAutomationTest\node_modules\@wdio\utils\build\shim.js:131:29)
                    at async Context.&lt;anonymous> (E:\Projects\Test-mobile\MobileAutomationTest\test\specs\example.e2e.js:26:3)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>